### ISSUE TITLE
"Brave Web Store" in brave://extensions

### ISSUE THREAD

**Issue by fmarier** [brave/brave-browser#24961]
We're showing "Brave Web Store" instead of "Chrome Web Store" in `brave://extensions`:
![Screenshot from 2022-08-24 16-03-13](https://user-images.githubusercontent.com/167821/186538639-222d2256-d473-4d30-8f30-9066803a1f17.png)

Affects Stable, Beta and Nightly:
```
Brave | 1.44.63 Chromium: 104.0.5112.102 (Official Build) nightly (64-bit)
Revision | 8e5396254975ef939f2ef7d0bd334e48a052b536-refs/branch-heads/5112@{#1478}
OS | Linux
```

**Comment by rebron** [comment]
cc: @mkarolin

**Comment by rebron** [comment]
@fmarier I'm not seeing this on other platforms. Linux only?

**Comment by fmarier** [comment]
Ah, could be Linux only. I have not tested on other platforms.

**Comment by rebron** [comment]
Looks like we have another location to grab in the Extensions themselves.
`https://www.reddit.com/r/brave_browser/comments/xsrqle/i_didnt_know_brave_team_launched_brave_web_store/`
![hsknlgt566r91](https://user-images.githubusercontent.com/5951041/193890824-30311d2c-8a01-42f5-8629-9ff680cf0bd5.png)

**Comment by spylogsster** [comment]
checked om desktops, unable to reproduce on master

**Comment by fmarier** [comment]
@spylogsster Did you check on Linux? I can still reproduce on Nightly 1.49.42 Chromium: 109.0.5414.87 (Official Build) nightly (64-bit).

**Comment by spylogsster** [comment]
<img width="929" alt="image" src="https://user-images.githubusercontent.com/2965009/215251211-e82faa86-157f-46d0-b356-37ae2461c9ab.png">

My Linux, v1.49.70

**Comment by rebron** [comment]
@fmarier I see what @spylogsster posted on 1.48.164 in current release and an older version of nightly 1.49.24.

**Comment by fmarier** [comment]
Yeah, we've not figured out yet why I'm seeing "Brave Web Store" on my Linux machines but nobody else is...

**Comment by bsclifton** [comment]
Seems to be a translation problem - cc: @mkarolin

**Comment by kjozwiak** [comment]
Removing the above from the `1.49.x` milestone as there's no PR associated with this and is not labelled as `release/blocking`. Once we figure out the above and have a fix, it can land in the associated milestone. @mkarolin if the above gets resolved with the `1.49.x` translations, we can move this back to `1.49.x` for QA to check/verify.

**Comment by fmarier** [comment]
Seems fixed in 1.70.94 Chromium: 128.0.6613.114 (Official Build) beta (64-bit) :
![Screenshot from 2024-09-04 12-41-21](https://github.com/user-attachments/assets/31ff3c15-6cc1-4c87-8bf3-e8a57342dd86)

**Comment by bsclifton** [comment]
Thanks for retesting, @fmarier! Does indeed look fixed 😄
